### PR TITLE
Add active mentorship playbook

### PR DESCRIPTION
**Pull request by renatoruk**

### CODE REVIEW

File: playbooks/active-mentorship.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+mentorship.
+
+To provide structure into the mentorship, we follow a daily routine sessions in which the mentee can ask their
+mentor questions and discuss given topics.
+
+The routine can be practiced as follows:
+
+-   keep the mentoring session short and focused
+-   maximum time is 30 minutes
+-   no minimum time is specified, meaning the session does not have to be help if there are no topics to discuss
```
Comment (ShockwaVee) [inline playbooks/active-mentorship.md:16]: Did you mean "the session does not have to **happen**"?

File: playbooks/active-mentorship.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Active mentorship
+description: Guide for practicing active mentorship between a mentor and mentee
+---
+
+Every junior team member has a dedicated mentor assigned to them for guidance, questions and general technical
```
Comment (ibrcko) [inline playbooks/active-mentorship.md:6]: I would like to suggest that not only junior devs are the ones that need mentorship. 

I think that there should be two categories, as we discussed on our weekly ITF meet:
a) Mentorship for less experienced developers (mentee has a dedicated mentor)
b) Mentorship for more experienced developers - mentee and mentor relation is dynamic: mid (mentor) -> mid, mid (mentor) -> sen

What do you think?

File: playbooks/active-mentorship.md (hunk 1 above)
Comment (krukru) [reply playbooks/active-mentorship.md:16]:
```suggestion
-   no minimum time is specified, meaning the session does not have to happen if there are no topics to discuss
```

File: playbooks/active-mentorship.md (hunk 2 above)
Comment (krukru) [reply playbooks/active-mentorship.md:6]: I would add to the doc that a junior developer MUST have a dedicated mentor, while any other level MAY request one. Of course the dynamic will be different, more experienced level developers will need less and different mentoring, the topics discussed will be more abstract (or highly specific, technology for example), etc.

I would for now focus on the mentorship program for junior level developers, where the program is most needed and the return value is highest (the cost of leaving junior developers on their own is highest). Do you agree @ibrcko?

Comment (ibrcko) [reply playbooks/active-mentorship.md:6]: I agree!

Can you please elaborate on what does "I would focus" mean?

File: playbooks/active-mentorship.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+To provide structure into the mentorship, we follow a daily routine sessions in which the mentee can ask their
+mentor questions and discuss given topics.
+
+The routine can be practiced as follows:
+
+-   keep the mentoring session short and focused
+-   maximum time is 30 minutes
+-   no minimum time is specified, meaning the session does not have to be help if there are no topics to discuss
+-   the mentee should prepare specific questions
```
Comment (napravicukod) [inline playbooks/active-mentorship.md:17]: + mentee can track discussed topics, conslusions and ideas inside a dedicated mentor-mentee doc

File: playbooks/active-mentorship.md (hunk 1 above)
Comment (renatoruk) [reply playbooks/active-mentorship.md:16]: I thought  "the session does not have to be held", but I mistyped that. What you're suggesting is definitely a better wording!

File: playbooks/active-mentorship.md (hunk 2 above)
Comment (renatoruk) [reply playbooks/active-mentorship.md:6]: You both said valid points and I agree that different mentorship categories exist. 

In this specific document, I would also focus on the "active dedicated mentorship" that we follow in Bornfight. 

Meaning that, @ibrcko , your option A of mentorship is the topic that's covered in this document.

So maybe a better name for this playbook would be "Active mentorship - junior level"

File: playbooks/active-mentorship.md (hunk 3 above)
Comment (renatoruk) [reply playbooks/active-mentorship.md:17]: Awesome idea. I suggest that we add this line:
```suggestion
-   the mentee should prepare specific questions
-   the mentee can track discussed topics, conclusions and ideas inside a dedicated mentor-mentee doc, visible to both parties
```

Do you agree, @napravicukod ?

**Pull request merged**